Tools-Override/CloudTest.targets

```diff
@@ -429,7 +429,31 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="CompressRuntimeDir" Condition="'$(SkipArchive)' != 'true'">
+  <Target Name="CopyCoreCLRTestLauncherForUAPToRuntimeDir" Condition="'$(BuildingUAPVertical)' == 'true'">
```

Comment:
FYI this entire file is going to be deleted soon. This logic should be moved to upload-tests.proj